intro-Linux/ch_filesystems.tex

```diff
@@ -46,26 +18,16 @@ \subsubsection{VO storage}
 
 \subsection{Quota}
 
-Space is limited on the cluster's storage, you can check your quota with the
-``show\_quota'' command:
-
-\begin{prompt}
-%\shellcmd{show\_quota}%
-VSC_HOME: used 1.89 GiB (66%\%%) quota 2.85 GiB (3 GiB hard limit)
-VSC_DATA: used 0 B (0%\%%) quota 23.8 GiB (25 GiB hard limit)
-VSC_SCRATCH_DELCATTY: used 0 B (0%\%%) quota 23.8 GiB (25 GiB hard limit)
-\end{prompt}
-
-Quota for personal directories is limited to 3GiB (home) and 25GiB (data \&
-scratch); for VO directories, it depends on the particular VO.
+Space is limited on the cluster's storage. To check your quota, see
+\href{\HPCManualURL#predefined-quotas}{Chapter 6, section titled "Pre-defined quota" of the HPC manual}.
 
-To figure out where you're quota is being spent, the ``du'' command can come in
-useful:
+To figure out where you're quota is being spent, the \verb|du| (\strong{d}isk \strong{u}sage)
+command can come in useful:
 
 \begin{prompt}
 %\shellcmd{du -sh test}%
 59M   test
 \end{prompt}
 
-Note: running ``du'' can take a while in case you have a lot of files and
+Note: running \verb|du| can take a while in case you have a lot of files and
 directories; use it with caution!
```

Comment:
Maybe we should extend this a bit, by mentioning:

```
Do \epmh{not} (frequently) run `du` on directories where large amounts 
of data are stored, since that will i) take a long time, ii) result in increased
load on the shared storage since (the metadata of) every file in those directories
will have to be inspected.
```